src/components/organisms/settings/organization/createNewOrganizationModal.tsx

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+
+import { useTranslation } from "react-i18next";
+
+import { ModalName } from "@enums/components";
+import { useModalStore } from "@src/store";
+
+import { Button } from "@components/atoms";
+import { Modal } from "@components/molecules";
+
+export const CreateNewOrganizationModal = () => {
```

Comment:
I'm sorry, I didn't understand correctly the last time what this modal's purpose is :(
Now I see.
And the first name was correct...
```suggestion
export const OnOrganizationCreatedModal = () => {
```
or
```suggestion
export const OrganizationCreatedModal = () => {
```
or
```suggestion
export const OrganizationPostCreationModal = () => {
```

Please change the filename as well.

Reply:
changed as `OrganizationCreatedModal`